test/JDBC/upgrade/latest/schedule

```diff
@@ -421,3 +421,4 @@ BABEL-3474
 dateadd_internal_df
 datetime2fromparts-after-15-2
 timefromparts
+sys_sysusers_dep
```

Comment:
What's the result of add sys_sysusers_dep into other mvu schedule ?

Reply:
I think we are only introducing this catalog in latest version, so I did not add to other upgrade schedule